aea/crypto/ledger_apis.py

```diff
@@ -45,27 +31,38 @@
 from aea.exceptions import enforce
 
 
-DEFAULT_LEDGER_CONFIGS = {
-    CosmosApi.identifier: {
+COSMOS_DEFAULT_ADDRESS = "INVALID_URL"
+COSMOS_DEFAULT_CURRENCY_DENOM = "INVALID_CURRENCY_DENOM"
+COSMOS_DEFAULT_CHAIN_ID = "INVALID_CHAIN_ID"
+ETHEREUM_DEFAULT_ADDRESS = "http://127.0.0.1:8545"
+ETHEREUM_DEFAULT_CHAIN_ID = 1337
+ETHEREUM_DEFAULT_CURRENCY_DENOM = "wei"
+FETCHAI_DEFAULT_ADDRESS = "https://rest-agent-land.fetch.ai"
+FETCHAI_DEFAULT_CURRENCY_DENOM = "atestfet"
+FETCHAI_DEFAULT_CHAIN_ID = "agent-land"
```

Comment:
Maybe it's better to remove the defaults entirely.

Reply:
yes

Reply:
Wait, this are used in `LedgerApis.ledger_api_configs`, which are important for ledger configuration overrides... since it is not necessary to make the PR to work, I suggest to address this in a separate PR.

Reply:
it also requires changes on the `LedgerApis` method parameters.

To elaborate, we have three types of ledger configurations:
- the default one, defined by the crypto plugin;
- the one in agent config overrides;
- the one provided as "hard override" as an argument of methods.

The last one is not necessary, but I think desirable, and requires changes to the `LedgerApis` methods.